docs/how-to-guides/develop-a-prompty/index.md

```diff
@@ -0,0 +1,384 @@
+# Prompty
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+Promptflow provides the `prompty` feature to provide customers with a simple way to develop prompt template.
+
+## Create a prompty
+In promptflow, a file with a `.prompty` extension will be treated as a prompty.
+
+### Prompty specification
+The prompty asset is a markdown file with a modified front matter. 
+The front matter is in yaml format that contains a number of metadata fields which defines model configuration and expected inputs of the prompty.
+
+Fields in the front matter:
+
+| Field       | Description                                                                             |
+|-------------|-----------------------------------------------------------------------------------------|
+| name        | The name of the prompt.                                                                 |
+| description | A description of the prompt.                                                            |
+| model       | Model configuration of prompty, contains connection info and LLM request parameters.    |
+| inputs      | Define the inputs received by the prompt template.                                      |
+| outputs     | Specify the fields in prompty result. (Only works when response format is json_object.) |
+| sample      | A dict or json file of inputs/outputs sample data.                                      |
+
+```yaml
+---
+name: Basic Prompt
+description: A basic prompt that uses the GPT-3 chat API to answer questions
+model:
+    api: chat
+    configuration:
+      type: azure_openai
+      azure_deployment: gpt-35-turbo
+      connection: azure_open_ai_connection
+    parameters:
+      max_tokens: 128
+      temperature: 0.2
+inputs:
```

Comment:
Shall we omit inputs outputs for now? As we are still finalizing the spec?
Or we should update after that?

Reply:
remove inputs/output in docs